### ISSUE TITLE
Both triggers are setted into the same axis

### ISSUE THREAD

**Issue by hik4ru** [irungentoo/Xiaomi_gamepad#15]
Hi,

I have a problem, into the Xbox gamepad propierties I can see how both triggers are setted into the same axis, making that if I press for example both triggers, it turns back to position zero. Could it be resolved with an update?

Thank you.

**Comment by irungentoo** [comment]
Pretty sure that's normal, try them in a game and you'll see them working perfectly.

**Comment by hik4ru** [comment]
No, it doesn't work perfectly, it keeps showing that it's the same axis in the game

**Comment by hik4ru** [comment]
Hi again,

I've seen that in Xinput-compatible games like Dolphin Emulator it seems to work ok, but in DirectInput games like old ones (e.g.: Grand Prix 4) it just recognize both triggers as one axis. Could this be fixed in DirectInput?

**Comment by Ryochan7** [comment]
That is working as designed. The DirectInput counterpart for an XInput gamepad has both triggers mapped to a single axis. I don't know if ScpVBus would allow using one of the alternative mappings instead of using the standard for XINPUT_DEVSUBTYPE_GAMEPAD.

https://msdn.microsoft.com/en-us/library/windows/desktop/hh405052(v=vs.85).aspx

**Comment by hik4ru** [comment]
My 360 and One pads work with 2 different axis; I don't know how work into the API (I'm only a beginner Java dev) but I know how it shows and works in the games. Tomorrow I'll upload a few screenshots for you to look at what I mean

**Comment by hik4ru** [comment]
http://imgur.com/a/WPyhZ

Here you have. As you can see on the first pic, the Xbox One controller has its triggers mapped as axis Z and IRZ, but Xiaomi's one is mapped only as Z, turning to position zero if both triggers are pressed

**Comment by irungentoo** [comment]
Just to be sure did you install the scp bus thing from my zip or from somewhere else?

**Comment by hik4ru** [comment]
Yes, I installed it. Before the scp from your zip I also tried with a DS3/4 app which internet said it works (I installed both so, I don't know it this could be a problem), but well... As I said, it's weird that my original pads works ok and this have that problem where it should be working exactly the same.

**Comment by hik4ru** [comment]
Hi?
